### PR TITLE
[hi] Add  content/hi/docs/tasks/tools/included/optional-kubectl-confi…

### PR DESCRIPTION
**Pull request by ashish-jaiswar**
### ### added one file 
localized optional-kubectl-configs-bash-linux.md to Hindi.
[issues](https://github.com/kubernetes/website/issues/30569)

### CODE REVIEW

File: content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+title: "Linux पर bash auto-completion करे"
+description: "Linux पर bash auto-completion के लिए कुछ वैकल्पिक विन्विन्यास।"
+headless: true
+---
+
+ ## परिचय
+
+ बैश के लिए kubectl  कम्प्लिशन स्क्रिप्ट `kubectl completion bash` कमांड के साथ उत्पन्न की जा सकती है।                         आपके शेल में  कम्प्लिशन  स्क्रिप्ट को सोर्स करने से kubectl ऑटोकम्प्लिशन ऐनेबल हो जाती है।
```
Comment (verma-kunal) [inline content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md:9]: it is recommended to follow a uniform localising style throughout the doc. 
for example: if we are using `bash` as it is in a certain sentence, its recommended to use the same pattern throughout!

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]
These are shortcode and should not be translated. This was also the reason netlify tests failed as netlify couldn't build the site.

As the log says
```
7:57:48 PM: ────────────────────────────────────────────────────────────────
7:57:48 PM:   1. build.command from netlify.toml                            
7:57:48 PM: ────────────────────────────────────────────────────────────────
7:57:48 PM: ​
7:57:48 PM: $ git submodule update --init --recursive --depth 1 && make deploy-preview
7:57:48 PM: hugo --enableGitInfo --buildFuture -b https://deploy-preview-30555--kubernetes-io-main-staging.netlify.app
7:57:48 PM: Start building sites …
7:57:48 PM: hugo v0.87.0-B0C541E4+extended linux/amd64 BuildDate=2021-08-03T10:57:28Z VendorInfo=gohugoio
7:57:51 PM: Error: Error building site: "/opt/build/repo/content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md:46:1": failed to extract shortcode: template for shortcode "ध" not found
7:57:51 PM: Total in 3161 ms
7:57:51 PM: Makefile:38: recipe for target 'deploy-preview' failed
7:57:51 PM: make: *** [deploy-preview] Error 255
```

**Comment by ashish-jaiswar** [comment]
@verma-kunal , @anubha-v-ardhan I updated the file please merge this PR.

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]
lgtm to me.

However, please could you squash & merge the number of commits to 1?

File: content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+source /usr/share/bash-completion/bash_completion
+```
+
+अपना शेल पुनः लोड करें और सत्यापित करें कि `type _init_completion` टाइप करके bash-completion सही ढंग से इंस्टॉल है।
+
+## kubectl ऑटोकम्प्लिशन ऐनेबल करे
+
+अब आपको यह सुनिश्चित करने की आवश्यकता है कि Kubectl  कम्प्लिशन स्क्रिप्ट आपके सभी शेल सत्रों (sourced) में प्राप्त हो जाए। आप ऐसा दो तरीकों से कर सकते हैं:
+
+- अपनी `~/.bashrc` फ़ाइल में  कम्प्लिशन स्क्रिप्ट सॉर्स करें :
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md:30]:
```suggestion
- अपनी `~/.bashrc` फ़ाइल में कम्प्लिशन स्क्रिप्ट सॉर्स करें:
```

File: content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    ```
+    
+
+यदि आप के पास kubectl के लिए एक अन्य नाम (alias) है, तो आप उस अन्य नाम के साथ काम करने के लिए शेल कम्प्लिशन को बढ़ा सकते हैं:    
+```bash
+echo 'alias k=kubectl' >>~/.bashrc
+echo 'complete -F __start_kubectl k' >>~/.bashrc
+```
+{{< note >}}
+बैश- कम्प्लिशन स्रोत सभी कम्प्लिशन स्क्रिप्ट `/etc/bash_completion.d` में ।
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md:47]:
```suggestion
bash-completion सभी कम्प्लिशन स्क्रिप्ट को `/etc/bash_completion.d` में सोर्स करता है।
```

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]
Final Review.
Please commit the suggestions if you agree with them.

**Comment by ashish-jaiswar** [comment]
hey, @anubha-v-ardhan I committed the suggestion, what is the next step?

**Comment by anubha-v-ardhan** [comment]
@ashish-jaiswar Looks like you merged the main branch along with the suggestions too which has caused 1011 commits on this PR. Can you fix it?

**Comment by ashish-jaiswar** [comment]
@anubha-v-ardhan How can I resolve this issue? can you suggest to me I am a new contributor?

**Comment by anubha-v-ardhan** [comment]
@ashish-jaiswar afaik, you can do one of the following to fix this (whichever will be easier for you)

1. Do a rebase and drop the unwanted commits.
2. Start a new branch off `dev-1.22-hi.1`, copy the text from here and make a new PR.

Its totally fine to make a new PR, just close this PR mentioning the new one if you choose the 2nd option

**Comment by ashish-jaiswar** [comment]
opening new PR

**Comment by jihoon-seo** [comment]
/remove-area blog release-eng
/remove-language de en es fr id it ja ko pl pt ru uk vi zh
/remove-sig release

**Pull request closed without merging**